### ISSUE TITLE
Installation ERROR: No matching distribution found for omnipath>=1.0.4

### ISSUE THREAD

**Issue by genecell** [theislab/squidpy#375]
Hi,

During installation of squidpy, I could not install omnipath=1.0.4, which is required by squidpy. Although I found the latest version of omnipath is 1.0.4 in https://pypi.org/project/omnipath/1.0.4/, I could not download  it via `pip install omnipath==1.0.4`.

The error:
```
pip install squidpy==1.0.1
Collecting squidpy==1.0.1
  Using cached squidpy-1.0.1-py3-none-any.whl (114 kB)
ERROR: Could not find a version that satisfies the requirement omnipath>=1.0.4 (from squidpy) (from versions: 1.0.0, 1.0.1, 1.0.2, 1.0.3)
ERROR: No matching distribution found for omnipath>=1.0.4
```

Thanks,
Min

**Comment by genecell** [comment]
I solved this problem by updating Python 3.6 to Python 3.7. I don't know whether this is the key.

**Comment by giovp** [comment]
@genecell either that or there was an old cached version of either of the two packages. Glad that it's solved now. New Scanpy release will loose support for 3.6 and we will modify accordingly reqs for Squidpy. Will close, feel free to reopen if necessary/unsolved
